src/components/HardwareListing/HardwareListing.tsx

```diff
@@ -43,6 +43,24 @@ interface Filter {
 const ModuleGroup = (props: { level: number; group: ListingGroup; hideGroup?: boolean }) => {
   const shipSnapShot = React.useContext(ShipSnapshotContext);
 
+  const onItemDragStart = React.useCallback(
+    (
+      typeId: ListingItem["typeId"],
+      slotType: ListingItem["slotType"],
+    ): ((e: React.DragEvent<HTMLDivElement>) => void) => {
+      return (e: React.DragEvent<HTMLDivElement>) => {
+        // TODO: In case if image was not loaded/cached before, when user will drag a module for the first time
+        // image wont be visible. On every subsequent drag of the same module image will be visible because it was already cached.
```

Comment:
A solution for this would be to load the image when hovering over the item. That might be wasteful in some cases, but helps solve this problem.

But for now, I won't do that in this PR. And also remove the `TODO`, as that will get lost in the woods. Maybe after merging this PR we can make an issue for it, so we are reminded that we should solve this.

Reply:
Another idea is to use one generic image for all modules, it wont be as pretty, but at least consistent. Or just module name instead of image